htdocs/core/menus/standard/eldy.lib.php

```diff
@@ -2064,6 +2064,9 @@ function get_left_menu_products($mainmenu, &$newmenu, $usemenuhider = 1, $leftme
 			$newmenu->add("/product/index.php?leftmenu=service&amp;type=1", $langs->trans("Services"), 0, $user->hasRight('service',  'read'), '', $mainmenu, 'service', 0, '', '', '', img_picto('', 'service', 'class="pictofixedwidth"'));
 			$newmenu->add("/product/card.php?leftmenu=service&amp;action=create&amp;type=1", $langs->trans("NewService"), 1, $user->hasRight('service',  'creer'));
 			$newmenu->add("/product/list.php?leftmenu=service&amp;type=1", $langs->trans("List"), 1, $user->hasRight('service',  'read'));
+			if (isModEnabled('Stock') && $conf->global->STOCK_SUPPORTS_SERVICES) {
```

Comment:
Can you replace $conf->global->STOCK... with
getDolGlobalString('STOCK...')

Reply:
@eldy  I think I made the change you requested.

FYI: I also did a grep for '$conf->global->STOCK_SUPPORTS_SERVICES' and it is mostly (37 files)
`    empty($conf->global->STOCK_SUPPORTS_SERVICES)
`
but also one like mine:
./htdocs/admin/stock.php:
`
print $form->selectarray("STOCK_SUPPORTS_SERVICES", $arrval, $conf->global->STOCK_SUPPORTS_SERVICES);
`
showed up in a total of 38 other files - maybe all those they should be fixed too?

Reply:
Yes, we should have other ref fixed.